Create DAS_reviewer_protocol.md (#36)
worldbank/wb-reproducible-research-repository: DAS_reviewer_protocol.md

---

## Step 1: Check that a DAS exists

The package must include a DAS. If there is no DAS and no data documentation of any kind, return the package to the authors before proceeding further.

---

## Step 2: Check the opening statement and Statement of Rights

### Opening statement [Recommended]

The DAS should begin with an overall categorization of the data used. If it is present, check that it correctly reflects the data in the package, i.e., that the accessibility level claimed (e.g., "all data are publicly available") actually matches the datasets used. The statement should indicate one of the following:

- All data are publicly available
- Some data cannot be made publicly available
- No data can be made publicly available
### Statement about Rights [Required]

A [Statement about Rights](https://github.com/worldbank/wb-reproducible-research-repository/blob/update_readme/resources/README_Template.md#statement-about-rights) is required in every DAS. It should confirm that the authors had legitimate access to the data and address redistribution rights, i.e., whether they have permission to share the data, or, if not, why not and under what terms others can access it.

> If this statement is missing, flag it as a required addition. 

Reviewer: ```suggestion
If this statement is missing, flag it as a required addition. The replicator can add this line directly if the README is in an editable format (Word, .md) and tell the authors that this change to the README was implemented.
```

If the data has not yet been deposited, inform the authors that this is required. They have the option to deposit as internal/restricted, publication does not mean the data must be made public. 

If the data is forthcoming or pending deposit, you can proceed with verification but must return to update the metadata entry once the data is published.
 
**Exception:** Data deposit is not required if the dataset was created by the authors specifically for this study using public sources, and has no value or use outside of the study itself.
 
#### Additional requirements for third-party unpublished or restricted data

If any dataset is restricted or confidential, the DAS must also include enough information for a reader to understand how they could request the same data along with a citation.

---

## Step 4: Check data accessibility and verify access

Each dataset falls into one of the following accessibility categories. For each type, check the DAS and verify access as described.

### Open data
*Data is publicly available and included in the reproducibility package.*

- Check that the link in the DAS works and that the data actually exists at that location.

> - You do not need to verify every value, but download the data to confirm that what is downloaded matches what is in the package.

Reviewer: Do we all actually do this?

- If the instructions for accessing the data are unclear, ask the authors to clarify.

> ### Accessible data

Reviewer: we should add a note here on data that is too time-consuming to review, like UNCOMTRADE, impossible to download, and check the data comparison tool. In this case, what we can do is check a few values and see it match and write in the report accordingly.

> - You do not need to verify every value, but download the data to confirm that what is downloaded matches what is in the package.

Author: ```suggestion
- Follow the access instructions in the DAS and verify that they are sufficient to download the data, i.e., that a third party could retrieve the data without any additional information. If the instructions are unclear or incomplete, ask the authors to clarify.
```

*Data is publicly available but does not allow redistribution and it is not included in the reproducibility package.*

- Follow the same steps as for Open data.

> - Additionally, run the [Dataset Comparison Tool](https://github.com/worldbank/wb-reproducible-research-repository/tree/main/resources/compare_datasets) to compare the publicly available version with the data provided by the authors.

Author: ```suggestion
- Additionally, run the [Dataset Comparison Tool](https://github.com/worldbank/wb-reproducible-research-repository/tree/main/resources/compare_datasets) to compare the publicly available version with the data provided by the authors. For datasets that are too large or time-consuming to download in full (e.g., UN Comtrade), running the comparison tool may not be feasible. In these cases, manually check a few key values against the source and document the approach taken in the verification report.
```

- If differences are due to periodic updates to the source data, note this in the verification report. If differences appear to be due to manual changes, confirm with the authors whether any modifications or processing were done. If processing was done in code, ask for the code. If it was done manually, ask the authors to document the steps. In general, encourage authors to start from the raw source data where possible.

### Limited-access data
*Data access requires purchase or human approval and is not included in the reproducibility package.*

- If the data is reasonably easy to access, follow the same steps as for Accessible data above.
- If access is difficult to obtain, document what was verified and flag any limitations in the report.

### Forthcoming data
*Data is forthcoming in the World Bank Microdata Library or Development Data Hub.*

- No access verification is needed at this stage.

> - Add a note to the GitHub issue to return to this once the data is published: update the metadata entry with the published link, and if the data is Accessible or Limited-access, run the comparison tool at that point.

Reviewer: ```suggestion
- Add the label `pending in RRR: data publication` to the GitHub issue to return to this once the data is published: update the metadata entry with the published link, and if the data is Accessible or Limited-access, run the comparison tool at that point.
```

### Restricted data
*There is no documented public access path, or data access was granted directly to the authors under a custom license that does not allow redistribution; the data is not included in the reproducibility package.*

- Check that the DAS contains enough information for a user to understand where the data comes from and how it can be requested.

---

## Common issues to flag

- **No DAS at all.** Return the package; this is a blocker.
- **DAS covers only some datasets.** Use the hash tool to confirm all input datasets are listed.
- **Package starts from an intermediate file** (e.g., `cleaned_for_analysis.dta`). This is a blocker. Return to authors with an explanation of the policy.
- **Opening statement does not match the actual data** (e.g., claims all data is public when some is restricted). Flag for correction.
- **Statement of Rights is missing.** Required in every DAS.
- **URL is missing for publicly available data.** Required if the data is accessible online.

> - **Access date is missing.** Required.

Reviewer: ```suggestion
- **Access date is missing.** Required. Verification can proceed without an access date, but request the details from the authors for the publication stage. 
```

> - **No license or redistribution statement for restricted data.** Required.

Reviewer: ```suggestion
- **No license or redistribution statement for restricted data.** Required. Verification can proceed without a license, but request the details from the authors for the publication stage. 
```

- **World Bank-owned data not deposited in MDL or DDH.** Flag and initiate deposit process (unless the exception applies).
- **Data only accessible via a staff member's personal storage.** Flag as a critical issue; data must be in a permanent, institutional location.
- **Restricted data has no access instructions.** A third party must be able to understand how to request the same data.

---

## Quick reference checklist

- [ ] DAS is present and clearly labeled
- [ ] Opening statement (if present) correctly reflects the data in the package
- [ ] Statement of Rights is included
- [ ] All input datasets are identified (use hash tool if needed)
- [ ] Each dataset is documented with required fields (filename, source, URL if available, access date, citation, license/redistribution terms, access instructions)
- [ ] License terms verified for each dataset

> - [ ] World Bank-owned data is deposited in MDL or DDH, or flagged as forthcoming (with GitHub issue note)

Reviewer: ```suggestion
- [ ] World Bank-owned data is deposited in MDL or DDH, or flagged as forthcoming (with label in the github issue included)
```
